lib/reporting/reports/suppliers/base.rb

```diff
@@ -40,10 +40,10 @@ def columns
             product:,
             variant_unit_name:,
             quantity:,
-            total_excl_vat_and_fees:,
+            total_excl_fees_and_tax:,
             total_excl_vat:,
```

Comment:
It's the existing locale and being used in many reports that's why it's being kept the same. We can create a small refactor tech debt to fix this across the app.